wofs/wofs_app.py

```diff
@@ -456,30 +464,47 @@ def estimate_job_size(num_tasks):
 @click.group(help='Datacube WOfS')
 @click.version_option(version=__version__)
 def cli():
+    """
+    Instantiate a click.group object
+    :return: None
+    """
     pass
 
 
 @cli.command(name='list', help='List installed WOfS config files')
 def list_configs():
-    for cfg in CONFIG_DIR.glob('*.yaml'):
+    """
+     List installed WOfS config files
+    :return: None
+    """
+    # Wofs config directory is accessed by providing the logical
+    # ancestors (dea/<date>) of the ROOT_DIR path.
+    for cfg in ROOT_DIR.parents[2].glob('wofs/config/*.yaml'):
```

Comment:
This looks much cleaner.

However it's not clear why you would go two levels above the ROOT_DIR. Typically ROOT would be the top